src/components/organisms/deployments/sessions/viewer.tsx

```diff
@@ -133,6 +133,11 @@ export const SessionViewer = () => {
 		};
 	}, [sessionInfo, formatTimeDifference]);
 
+	const visibleRefreshButton = useMemo(
+		() => sessionInfo?.state === SessionState.created || sessionInfo?.state === SessionState.running,
+		[sessionInfo?.state]
+	);
```

Comment:
In the future, we will have "Pending" status.
So you better do this:
```suggestion
	const visibleRefreshButton = useMemo(
		() => sessionInfo?.state !== SessionState.completed || sessionInfo?.state !== SessionState.error || sessionInfo?.state !== SessionState.stopped,
		[sessionInfo?.state]
	);
```

Reply:
Also only disable the button, don't hide it.

Reply:
changed